flink-table/flink-table-planner/src/test/java/org/apache/flink/table/api/CompiledPlanITCase.java

```diff
@@ -83,14 +83,25 @@ void testCompilePlanSql() throws IOException {
                 tableEnv.compilePlanSql("INSERT INTO MySink SELECT * FROM MyTable");
         String expected = TableTestUtil.readFromResource("/jsonplan/testGetJsonPlan.out");
         assertThat(
-                        TableTestUtil.replaceExecNodeId(
-                                TableTestUtil.replaceFlinkVersion(
-                                        TableTestUtil.getFormattedJson(
-                                                compiledPlan.asJsonString()))))
+                        getPreparedToCompareCompilePlan(
+                                TableTestUtil.getFormattedJson(compiledPlan.asJsonString())))
                 .isEqualTo(
-                        TableTestUtil.replaceExecNodeId(
-                                TableTestUtil.replaceFlinkVersion(
-                                        TableTestUtil.getFormattedJson(expected))));
+                        getPreparedToCompareCompilePlan(TableTestUtil.getFormattedJson(expected)));
```

Comment:
ideally here should be just expected however because of difference between `compiledPlan.asJsonString` and `expected` it is like it is
room for improvement within another PR

Reply:
done in hotfix commit